datalake/src/test/java/com/sequenceiq/datalake/controller/sdx/SdxControllerTest.java

```diff
@@ -190,6 +197,21 @@ void refreshDatahubs() {
         verify(sdxService, times(1)).refreshDataHub(SDX_CLUSTER_NAME, null);
     }
 
+    @Test
+    void validateBackupStorage() {
+        SdxCluster sdxCluster = getValidSdxCluster();
+        sdxCluster.setClusterName(SDX_CLUSTER_NAME);
+        SdxBackupLocationValidationRequest sdxBackupLocationValidationRequest = new SdxBackupLocationValidationRequest(SDX_CLUSTER_NAME);
+        when(storageValidationService.validateBackupStorage(any())).thenReturn(null);
+        when(sdxService.getByNameInAccount(USER_CRN, SDX_CLUSTER_NAME)).thenReturn(sdxCluster);
+
+        ThreadBasedUserCrnProvider.doAs( USER_CRN, () -> sdxController.validateBackupStorage(sdxBackupLocationValidationRequest));
+
+        verify(sdxService, times(1)).getByNameInAccount
+                (USER_CRN, SDX_CLUSTER_NAME);
```

Comment:
Please join these two lines. The opening parenthesis of the `getByNameInAccount()` invocation should be on the same line as the method name (CheckStyle error).